apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/StorageActivity.java

```diff
@@ -75,6 +81,24 @@ public void accept(PaginatedDocuments<TestDocument> documents) {
             }
         });
 
+        /* List the user documents. */
+        sUserDocumentList.clear();
+        mUserDocumentContents.clear();
+        if (mAdapterUser == null) {
+            String accountId = MainActivity.sSharedPreferences.getString(ACCOUNT_ID, null);
+            if (accountId != null) {
+                Storage.list(Constants.USER, Map.class).thenAccept(new AppCenterConsumer<PaginatedDocuments<Map>>() {
+                    @Override
```

Comment:
```suggestion
                    
                    @Override
```
blank lines before annotations